### DISCUSSION TITLE
Ready to benchmarks?

### DISCUSSION THREAD

**Discussion by iAxel** [confetti-framework/confetti#136] [Q&A] [answered]
Hello everyone!

Since the day of the announcement, I've been following the framework, will there be performance results?

Try to post results [here](https://github.com/the-benchmarker/website)

**Comment by reindert-vetter** [comment, marked as answer]
Interesting! I haven't made any performance results public yet. But feel free to publish it yourself. 😄

**Reply by iAxel** [reply]
Thanks for answer) Okay I will try)
